lib/dealership.rb

```diff
@@ -0,0 +1,44 @@
+require 'pry'
+class Dealership
+  attr_reader :inventory
+  def initialize(argument_1, argument_2)
+    @argument_1 = argument_1
+    @argument_2 = argument_2
+    @inventory = []
+  end
+
+  def inventory_count
+    @inventory.count
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    @inventory = []
+      false
+  end
```

Comment:
I think you wanted to check if @inventory was empty

```
def has_inventory?
  if (@inventory == [])
    return false
  end
  true
end
```

Another way to do it:
```suggestion
  def has_inventory?
    inventory_count > 0
  end
```